src/Tribe/RSVP.php

```diff
@@ -2273,33 +2299,41 @@ public function generate_tickets_for( $product_id, $ticket_qty, $attendee_detail
 	public function parse_attendee_details() {
 		$order_id = self::generate_order_id();
 
-		$attendee_email     = empty( $_POST['attendee']['email'] ) ? null : sanitize_email( $_POST['attendee']['email'] );
+		$first_attendee = [];
+
+		if ( isset( $_POST['tribe_ticket_attendee'] ) && ! empty( $_POST['tribe_ticket_attendee'] ) ) {
+			$first_attendee = $_POST['tribe_ticket_attendee'][0];
+		} elseif ( isset( $_POST['attendee'] ) ) {
+			$first_attendee = $_POST['attendee'];
```

Comment:
* :warning: Detected usage of a non-sanitized input variable: $_POST['attendee']
* :no_entry_sign: Processing form data without nonce verification.

<sub>hash: fe0135cb53090c261bd00a8defd41363</sub>